pyproject.toml

```diff
@@ -13,7 +13,7 @@ requires-python = ">=3.7"
 keywords = ["jupyter", "ypy"]
 dependencies = [
     "importlib_metadata >=3.6; python_version<\"3.10\"",
-    "y-py >=0.5.3,<0.6.0",
+    "y-py >=0.5.3,<0.5.5",
```

Comment:
Ypy v0.5.4 is likely to create other issues too.
I think v0.5.5 should now be yanked on PyPI, since all wheels are not created from the same code base (ARM-based wheels have https://github.com/y-crdt/ypy/pull/102 while others don't).
@Waidhoferj @dmonad @SylvainCorlay I really need to have release rights for `ypy` on PyPI, these issues are badly affecting Jupyter.